railties/lib/rails/tasks/framework.rake

```diff
@@ -48,7 +48,7 @@ namespace :app do
           require 'rails/generators'
           require 'rails/generators/rails/app/app_generator'
           gen = Rails::Generators::AppGenerator.new ["rails"],
-                                                    { api: !!Rails.application.config.api_only },
+                                                    { api: !!Rails.application.config.api_only, update: true, force: ENV['FORCE'] },
```

Comment:
I really don't like the idea of adding this option since end users won't need it.

The tests already created the AppGenerator manually, what's the harm in continuing that? 😁